config.json

```diff
@@ -24,41 +24,40 @@
     "home":"vac.md",
     "fileOrdersInSidenav": [
       "vac.md",
+      "principles.md",
       "research-areas.md",
-      "projects.md"
+      "projects.md",
+      "open-problems.md",
+      "contribute.md",
+      "media.md",
     ],
     "folderOrdersInSidenav": [
-      "research",
-      "about"
+      "research"
     ],
     "external_links": [
       {
-        "text": "Research & experimentation",
-        "url": "https://github.com/vacp2p/research/",
-        "category": "Resources"
-      },
-      {
-        "text": "Vac RFCs/Specs",
+        "text": "Specs/RFCs",
         "url": "https://rfc.vac.dev/",
         "category": "Resources"
       },
       {
-        "text": "Vac Forum",
+        "text": "Forum",
         "url": "https://forum.vac.dev/",
         "category": "Resources"
       },
       {
-        "text": "Waku v2 training session",
-        "url": "https://drive.google.com/file/d/19P3oDNXGBDClfcS6Sgp0t9LYr3UbIFGt/view",
+        "text": "Waku.org",
+        "url": "https://waku.org",
         "category": "Resources"
       }
+
     ]
   },
   "ds_id": "vac",
   "seo": {
-    "title": "Vac",
-    "description": "Vac is an R&D org creating modular p2p protocols for private, secure, censorship-resistant communication.",
-    "cc": "Vac researches peer-to-peer, private, censorship-resistant communication",
+    "title": "Vac Research",
+    "description": "Vac - Communication, Privacy, Etc.",
```

Comment:
After some more thought: I'd rather not use "etc.": unscientific, imo.

Especially in the projects descriptions below, I'd not use "etc" (see my further comments).

Reply:
Disagree, for several reasons.

1) It is a tag line, there's no "unscientific". If we mean not "Serious", this is a strength (counter signaling vs pretend play)

2) It is fun and playful, and acts as a good counter weight to the more serious nature of various security write-ups. It is unexpected and makes people laugh, which is good for standing out and people remembering

3) It is more truthful, because we can't capture everything in two or a few terms, so it accurate represents reality

Reply:
Just my opinion :).

Reply:
to me there's also a difference between deliberate "Etc" here and more haphazard "etc" in project descriptions, where being complete and enumerating is doable